nixos/modules/services/x11/hardware/libinput.nix

```diff
@@ -85,7 +91,7 @@ let cfg = config.services.xserver.libinput;
 
       middleEmulation = mkOption {
         type = types.bool;
-        default = true;
+        default = mkDefaultForDevice deviceType { };
```

Comment:
This options makes sense if your mouse is have only two buttons (I think this is the case even for some recent mouses, like Logitech MX Master series, where the scroll doesn't click). But setting to false the default since most mouses have three buttons.